includes/Maps_Mapper.php

```diff
@@ -193,6 +193,11 @@ private static function getEvenMawrCommonParameters() {
 			'default' => false,
 		];
 
+		$params['scrollwheelzoom'] = [
+			'type' => 'boolean',
+			'default' => true,
+		];
```

Comment:
Please add this to the leaflet specific parameters if it is only supported for that mapping service